delete_by_query.py

```diff
@@ -63,6 +63,17 @@ def delete_docs(es_url, index, doc_type, query):
     parser = argparse.ArgumentParser(
         description='Delete documents from ElasticSearch by query.'
     )
+
+    parser.add_argument(
```

Comment:
You might also be interested in these libraries - http://docs.python-guide.org/en/latest/scenarios/cli/